src/style.css

```diff
@@ -1 +1,6 @@
-/* add styles here */
+@import 'styles/page.css';
+@import 'styles/stars-1.css';
+@import 'styles/stars-2.css';
+@import 'styles/stars-3.css';
+@import 'styles/stars-4.css';
+@import 'styles/stars-5.css';
```

Comment:
While you have separated the styles into different files, the task requires that each BEM block's CSS file should be linked directly in the HTML file. Avoid using `@import` in a single `style.css` file. Instead, link each CSS file separately in the HTML.